pkg/controller/v1beta1/inferenceservice/components/predictor.go

```diff
@@ -298,7 +313,7 @@ func (p *Predictor) Reconcile(isvc *v1beta1.InferenceService) (ctrl.Result, erro
 	}
 	statusSpec, _ := isvc.Status.Components[v1beta1.PredictorComponent]
 	if rawDeployment {
-		podLabelValue = constants.GetRawServiceLabel(constants.DefaultPredictorServiceName(isvc.ObjectMeta.Name))
+		podLabelValue = constants.GetRawServiceLabel(constants.PredictorServiceName(isvc.ObjectMeta.Name))
```

Comment:
Should here use `predictorName` ?

Reply:
Yes @yuzisun . You're correct! Let me update.

Reply:
updated!